src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -120,6 +126,19 @@
         </div>
     {% endif %}
 
+    {% if collector.firewall|default([]) is not empty %}
+        <h2>Security Firewall</h2>
+
+        <div class="metrics">
+            <div class="metric">
+                <span class="value">{{ collector.firewall.name|default('unknown') }}</span>
+                <span class="label">Firewall Name</span>
```

Comment:
I will do my best for the bools as metrics :) 

About the request_matcher, I think you're right, it's not so useful, so I hesitate to don't store it in the config at all, or only when a custom one is set in the firewall.

I will make some tries about displaying the listeners, surely raw.